src/NLU.DevOps.Luis/LuisClient.cs

```diff
@@ -5,28 +5,18 @@ namespace NLU.DevOps.Luis
 {
     using System;
     using System.Collections.Generic;
-    using System.Net;
-    using System.Net.Http;
-    using System.Text;
     using System.Threading;
     using System.Threading.Tasks;
-    using Logging;
-    using Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring;
     using Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring.Models;
-    using Microsoft.Azure.CognitiveServices.Language.LUIS.Runtime;
     using Microsoft.Azure.CognitiveServices.Language.LUIS.Runtime.Models;
-    using Microsoft.CognitiveServices.Speech;
-    using Microsoft.CognitiveServices.Speech.Audio;
-    using Microsoft.CognitiveServices.Speech.Intent;
-    using Microsoft.Extensions.Logging;
-    using Newtonsoft.Json;
 
     internal class LuisClient : ILuisClient
     {
-        private const string Protocol = "https://";
-        private const string Domain = ".api.cognitive.microsoft.com";
+        public const string Protocol = "https://";
```

Comment:
I'd lean towards bumping these consts as private in the places they are used, or if there is strong case to have a single place in code to edit these, create a constants file. You are creating a circular dependency here between `LuisClient` and `LuisTrainingClient`/`LuisPredictionClient`

Reply:
Yeah good point, i can move it